File: app/src/main/java/com/joeshuff/dddungeongenerator/generator/monsters/SpecialAbility.java
package com.joeshuff.dddungeongenerator.generator.monsters;

import com.google.gson.annotations.SerializedName;

public class SpecialAbility {
    String name;
    String desc;

    @SerializedName("attack_bonus")
    int attackBonus;

    @SerializedName("damage_dice")
    String damageDice;

    @SerializedName("damage_bonus")
    int damageBonus;

    public String getName() {
        return name;
    }

    public String getDesc() {
        return desc;
    }

    public int getAttackBonus() {
        return attackBonus;
    }

    public String getDamageDice() {
        return damageDice;
    }

    public int getDamageBonus() {
        return damageBonus;
    }
}

File: app/src/main/java/com/joeshuff/dddungeongenerator/generator/monsters/SpecialAbility.kt
package com.joeshuff.dddungeongenerator.generator.monsters

import com.google.gson.annotations.SerializedName

class SpecialAbility {
    var name: String? = null
      get

    var desc: String? = null
      get

    @SerializedName("attack_bonus")
    var attackBonus = 0

    @SerializedName("damage_dice")
    var damageDice: String? = null

    @SerializedName("damage_bonus")
    var damageBonus = 0

}